### PR TITLE
use gene panel multiple study endpoint

### PR DESCRIPTION
**Pull request by adamabeshouse**

### CODE REVIEW

File: src/pages/resultsView/ResultsViewPageStore.ts [hunk 1]
```diff
@@ -483,43 +483,41 @@ export class ResultsViewPageStore {
...
-                        genePanelDataFilter:{
-                            entrezGeneIds: this.genes.result!.map(x=>x.entrezGeneId),
-                            sampleIds: dataQueryFilter.sampleIds,
-                            sampleListId: dataQueryFilter.sampleListId
-                        } as GenePanelDataFilter
+            const studyToMutationMolecularProfile = this.studyToMutationMolecularProfile.result!;
+            const sampleMolecularIdentifiers:SampleMolecularIdentifier[] = [];
+            this.samples.result!.forEach(sample=>{
+                const profile = studyToMutationMolecularProfile[sample.studyId];
+                if (profile) {
```
Comment (alisman) [inline src/pages/resultsView/ResultsViewPageStore.ts]: provide for case where there's no profile

Comment (adamabeshouse) [reply src/pages/resultsView/ResultsViewPageStore.ts]: if theres no profile, we simply dont add that sample to the query

Comment (alisman) [reply src/pages/resultsView/ResultsViewPageStore.ts]: but if sampleMolecularIdentifiers doesn't accumulate anything (as would happen with CNA only query?) then we'll call service with empty array?

Comment (adamabeshouse) [reply src/pages/resultsView/ResultsViewPageStore.ts]: oh oops i forgot to push that update

**Pull request merged**